alpine-consul/root/etc/services.d/consul-template/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv sh
```

Comment:
Also, perhaps the looping logic could be tidied up by changing this from a shell script to an execline script and utilising s6-svwait, http://skarnet.org/software/s6/s6-svwait.html. What do you think?